IglooSoftware/Challenge-Frontend-150
# Update code challenge 150

### What's changed?
Replaced the previous challenge with a new one using create-react-app

### How to test
See the README.md file and designs folder. If you want to see my solution, you can see branch IGLOO-15102-c which I'll delete later.
https://github.com/andrewlouie/Challenge-Frontend-150/compare/IGLOO-15102...andrewlouie:IGLOO-15102-C?expand=1
Pull and npm install. Make sure the linting works.
